practice/src/index.html

```diff
@@ -38,5 +38,69 @@
       </div>
     </div>
   </header>
+
+  <main class="main-section">
+    <!-- Best Place section -->
+    <section class="container place-section">
+      <div class="wrapper-place">
+        <img src="../src/assets/images/banner.webp" alt="" class="image-banner">
```

Comment:
Should use background image for both mobile, tablet, and desktop. Not only use background image for mobile